README.md

```diff
@@ -334,6 +334,80 @@ class Tracking < Coach::Middleware
 end
 ```
 
+# Coach CLI (alpha)
+
+_Note: The `coach` CLI is currently considered **alpha** stability, though we welcome bug
+reports if you find it doesn't work with your application._
+
+As well as the library, the Coach gem now comes with a command line tool - `coach`.
+
+When working in a large codebase that uses Coach, one of the challenges you may run into
+is understanding the `provide`/`require` graph made up of all the middleware chains you've
+built. While the library enforces the correctness of those chains at boot time, it doesn't
+help you understand those dependencies. That's where the `coach` CLI comes in!
+
+Currently, the `coach` CLI supports two commands.
+
+## `find-provider`
+
+`find-provider` is the simpler of the two commands. Given the name of a Coach middleware
+and a value that it requires, it outputs the name of the middleware that provides it.
+
+```bash
+$ bx coach find-provider HelloUser user
```

Comment:
I would fully qualify this. As an example, I use `alias be=bundle exec`, and there are many who won't have aliased and might wonder what this `bx` command is!

Reply:
Lol, massive copy-paste fail there.